https://bugs.gentoo.org/916258
https://github.com/audacity/audacity/discussions/5841
https://github.com/matoro/audacity/commit/270be88a9390eb25c2f4b16030f6897f08a6a685

From 270be88a9390eb25c2f4b16030f6897f08a6a685 Mon Sep 17 00:00:00 2001
From: matoro <matoro@users.noreply.github.com>
Date: Mon, 15 Jan 2024 20:21:08 -0500
Subject: [PATCH] Remove dependency on conan for unit tests

These run fine with catch2 installed from system package manager; there
is no reason to enforce conan dependency.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -228,15 +228,11 @@ cmd_option( ${_OPT}has_url_schemes_support
    "Build custom URL schemes support into Audacity"
    Off)
 
-include( CMakeDependentOption )
-
-cmake_dependent_option(
-   ${_OPT}has_tests
+cmd_option( ${_OPT}has_tests
    "Enables automated testing support"
-   On
-   "${_OPT}conan_enabled"
-   Off
-)
+   On)
+
+include( CMakeDependentOption )
 
 cmake_dependent_option(
    ${_OPT}has_audiocom_upload
